hack/app-sre/kustomization.yaml

```diff
@@ -0,0 +1,43 @@
+# This kustomize configuration deploys what is required for the hive-operator
```

Comment:
https://github.com/openshift/hive/blob/master/overlays/template/kustomization.yaml

why do we need to make a new one?

Reply:
there's also https://github.com/openshift/hive/blob/master/config/kustomization.yaml

Reply:
we already do make deploy to deploy using kustomize to our dev enviornment. how different is the app-sre that demands a completely separate step.

Reply:
Different enough :) Or I would have done it with the existing kustomizations.

The whole point of kustomize is to make it easy to do this kind of thing without needing multiple copies of the manifests.